src/ncp/posix/netif.cpp

```diff
@@ -57,10 +59,48 @@ otbrError Netif::Dependencies::Ip6Send(const uint8_t *aData, uint16_t aLength)
     return OTBR_ERROR_NONE;
 }
 
+otbrError Netif::Dependencies::Ip6MulAddrUpdateSubscription(const otIp6Address &aAddress, bool aIsAdd)
+{
+    OTBR_UNUSED_VARIABLE(aAddress);
+    OTBR_UNUSED_VARIABLE(aIsAdd);
+    return OTBR_ERROR_NONE;
+}
+
+#if NETIF_USE_MLD_MONITOR
+OT_TOOL_PACKED_BEGIN
+struct MLDv2Header
```

Comment:
Maybe it's a good chance to make it conforming to the OT's naming style. i.e. Mldv2Header

I'm fine with keeping it as-is, though.